application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -134,22 +136,31 @@ private void handleListCommand(SlashCommandInteractionEvent event) {
         long guildId = event.getGuild().getIdLong();
         long userId = event.getUser().getIdLong();
 
-        database
-            .read(context -> context.selectFrom(PENDING_REMINDERS)
-                .where(PENDING_REMINDERS.GUILD_ID.eq(guildId)
-                    .and(PENDING_REMINDERS.AUTHOR_ID.eq(userId)))
-                .orderBy(PENDING_REMINDERS.CREATED_AT.asc())
-                .fetch())
-            .forEach(reminder -> remindersEmbed.addField(reminder.getContent(),
-                    getDescription.apply(reminder.getChannelId(), reminder.getRemindAt()), false));
+        Result<PendingRemindersRecord> pendingReminders = getReminders(guildId, userId);
 
-        if (remindersEmbed.getFields().isEmpty()) {
+        if (pendingReminders.isEmpty()) {
             remindersEmbed.setDescription("No pending reminders");
+        } else {
+            pendingReminders.forEach(reminder -> {
+                String content = reminder.getContent();
+                long channelId = reminder.getChannelId();
+                Instant remindAt = reminder.getRemindAt();
+
+                remindersEmbed.addField(content, getDescription.apply(channelId, remindAt), false);
+            });
         }
 
         event.replyEmbeds(remindersEmbed.build()).setEphemeral(true).queue();
     }
 
+    private Result<PendingRemindersRecord> getReminders(long guildId, long userId) {
```

Comment:
getPendingReminders

also, id change it to accept Guild, User. with two longs, its easier for people to mess up and accidentally give them in the wrong order.